src/components/Icon/index.js

```diff
@@ -7,7 +7,10 @@ import * as Expensicons from './Expensicons';
 
 const propTypes = {
     // The asset to render.
-    src: PropTypes.oneOf(_.values(Expensicons)).isRequired,
+    src: PropTypes.oneOfType([
+        PropTypes.oneOf(_.values(Expensicons)),
+        PropTypes.func,
```

Comment:
This is because I was reluctant to add the Zoom and Meet logos to a file called `Expensicons` 😄. Willing to do so, but this was my preferred method.

Reply:
I would say just do it! We could just as easily have called it `IconLibrary` or something, but `Expensicons` is just a fun name. No need to let that affect the implementation imo.

Reply:
Good call just importing the full-color `svg` directly where you used it. 👍🏼
But I think that means that this change is unnecessary.

Reply:
Oh, I see why you need this. My mistake 👍

Reply:
Sorry I'm not seeing it where is this a `func` ?

Reply:
They are imported as functional components

Reply:
Ah yeah good point. I think that is redundant now since any function will also pass this type check